peewee_async/databases.py

```diff
@@ -28,6 +31,8 @@ def __init__(self, database: Optional[str], **kwargs: Any) -> None:
     async def aio_connect(self) -> None:
         """Creates a connection pool
         """
+        if self.deferred:
+            raise Exception('Cannot connect before db inited')
```

Comment:
Let's use this text: **Error, database must be initialized before creating a connection pool.**